tensorflow/java/src/main/java/org/tensorflow/Tensor.java

```diff
@@ -140,15 +129,14 @@
 
   /**
    * Create a Tensor of data type {@code dtype} from a Java object. Requires the parameter {@code T}
-   * to match {@code type}, but this condition is not checked statically. Use class {@link
-   * org.tensorflow.Tensors} to create Tensors in a fully type-safe way.
+   * to match {@code type}, but this condition is checked only at run time.
    *
    * @param obj the object supplying the tensor data.
    * @param dtype the data type of the tensor to create. It must be compatible with the run-time
    *     type of the object.
    * @return the new tensor
    */
-  public static Tensor<?> create(Object obj, DataType dtype) {
+  private static Tensor<?> create(Object obj, DataType dtype) {
```

Comment:
This is a new method as of this PR, so nothing is really lost by making it private.